cmd/snap/cmd_download.go

```diff
@@ -64,14 +76,32 @@ func init() {
 		"basename": i18n.G("Use this basename for the snap and assertion files (defaults to <snap>_<revision>)"),
 		// TRANSLATORS: This should not start with a lowercase letter.
 		"target-directory": i18n.G("Download to this directory (defaults to the current directory)"),
+		// TRANSLATORS: This should not start with a lowercase letter.
+		"direct": i18n.G("Do not try to connect to snapd to download"),
 	}), []argDesc{{
 		name: "<snap>",
 		// TRANSLATORS: This should not start with a lowercase letter.
 		desc: i18n.G("Snap name"),
 	}})
 }
 
-func fetchSnapAssertions(tsto *image.ToolingStore, snapPath string, snapInfo *snap.Info) (string, error) {
+func printInstallHint(assertPath, snapPath string) {
+	// simplify paths
+	wd, _ := os.Getwd()
+	if p, err := filepath.Rel(wd, assertPath); err == nil {
+		assertPath = p
+	}
+	if p, err := filepath.Rel(wd, snapPath); err == nil {
+		snapPath = p
+	}
+	// add a hint what to do with the downloaded snap (LP:1676707)
+	fmt.Fprintf(Stdout, i18n.G(`Install the snap with:
+   snap ack %s
+   snap install %s
+`), assertPath, snapPath)
+}
+
```

Comment:
We might want a TODO in this file about trying to share more code with and look at reorganizing image helpers. Notice that in this factoring of things ubuntu-image doesn't work like snap download.